modules/compute/container-registry/.bicep/nested_privateEndpoint.bicep

```diff
@@ -0,0 +1,60 @@
+param location string
```

Comment:
@Gordonby / @dciborow -- we should decide what we want the "modules" directory to be called. It's not user-facing, so it is not critical, but might as well be consistent

Reply:
Just a quick thought, and it is not strictly related to the name of the "modules" directory, however I was wondering if these sort of internal modules, could actually be published modules within the registry that we can reference as versioned modules instead of embedding as a file in each module? 

Hoping this would make maintenance of the modules easier in the future, especially as the catalog expands.

Happy to raise a new issue to propose the ideal formally if you folks think the idea is worth pursing? Thanks!